roles/sudoers/tasks/teardown.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+# Copyright 2021 Cloudera, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Remove the FreeIPA sudo rule
+- name: Remove sudo rule for passwordless sudo
+  community.general.ipa_sudorule:
+    ipa_user: "{{ sudoers__env_admin_username }}"
+    ipa_pass: "{{ sudoers__env_admin_password }}"
+    name: "{{ sudoers__sudorule_name }}"
+    state: absent
+
+# Remove the FreeIPA group for sudo
+- name: Remove sudoers group
+  community.general.ipa_group:
+    ipa_user: "{{ sudoers__env_admin_username }}"
```

Comment:
See above

Reply:
Fixed as part of https://github.com/cloudera-labs/cloudera.exe/pull/6/commits/488871c7128bd4a4f1b98b180929a4901d7851e0